x/programs/runtime/import_program.go

```diff
@@ -48,20 +53,17 @@ func NewProgramModule(r *WasmRuntime) *ImportModule {
 				newInfo.Params = parsedInput.Params
 				newInfo.Fuel = parsedInput.Fuel
 
-				result, err := r.CallProgram(
+				result := r.CallProgram(
```

Comment:
The logic to handle the trap error should exist here; or at least the decision to use said logic. Can you put `CallProgram` back to the way it was? 

If you want `func (r *WasmTime) ConcertErrorToResponse(err error) Response` or whatever, you can do that so that the logic is re-usable, but this is the "presentation" layer where the code should be making decisions about how the response should specifically be presented to the wasm-module.

Reply:
I have renamed the `callProgram` function to a more explicit `CallProgramToResponse` and kept the `CallProgram` function as-is.